crates/wasm/fs/metadata.rs

```diff
@@ -0,0 +1,44 @@
+use crate::fs::JsResult;
+use crate::value;
+use js_sys::{Object, Reflect};
+use wasm_bindgen::JsValue;
+use wnfs::{Metadata, UnixFsMetadata, UnixFsNodeKind};
+
+pub(crate) struct JsMetadata<'a>(pub(crate) &'a Metadata);
+
+impl TryInto<JsValue> for JsMetadata<'_> {
+    type Error = js_sys::Error;
+
+    fn try_into(self) -> JsResult<JsValue> {
+        let metadata = Object::new();
+        let unix_meta = unix_fs_to_js_value(&self.0.unix_fs)?;
+
+        Reflect::set(&metadata, &value!("unixMeta"), &unix_meta)?;
+        Reflect::set(&metadata, &value!("version"), &value!("3.0.0"))?;
```

Comment:
You missed the placeholder here.

```suggestion
        let version = value!(self.0.version.to_string());

        Reflect::set(&metadata, &value!("unixMeta"), &unix_meta)?;
        Reflect::set(&metadata, &value!("version"), &version)?;
```